vuu-ui/packages/vuu-data/src/server-proxy/server-proxy.ts

```diff
@@ -795,8 +813,8 @@ export class ServerProxy {
             const viewport = this.viewports.get(clientViewportId);
             if (viewport) {
               viewport.setTableMeta(body.columns, body.dataTypes);
-            } else {
-              logger.warn(
+            } else if (log.warnEnabled) {
```

Comment:
not sure if you've fully grokked the purpose of the code guards, you don't need to use them when logging a simple string.

https://commons.apache.org/proper/commons-logging/guide.html#Code_Guards